Doc/tutorial/venv.rst

```diff
@@ -44,7 +44,7 @@ whichever version you want.
 To create a virtual environment, decide upon a directory where you want to
 place it, and run the :mod:`venv` module as a script with the directory path::
 
-   python3 -m venv tutorial-env
+   python -m venv tutorial-env
```

Comment:
I think we should leave `python3` here as it is an instruction how to create a virtual environment. I didn't change it on purpose.